astropy/stats/sigma_clipping.py

```diff
@@ -165,6 +165,13 @@ class SigmaClip:
         an ``axis`` keyword to return an array with axis dimension(s)
         removed.  The default is ``'std'``.
 
+    grow : float or `False`, optional
+        Radius within which to mask the neighbouring pixels of those that
+        fall outwith the clipping limits (only applied along ``axis``, if
+        specified). A value of 1 will mask the nearest pixels in a cross
```

Comment:
```suggestion
        specified). As an example, for a 2D image a value of 1 will mask the nearest pixels in a cross
```

Reply:
Yes. The original description is also true for >2D images (if you interpret it as a higher-dimensional "cross") but I think your example makes it clear enough.